packages/flame/test/extensions/canvas_test.dart

```diff
@@ -1,9 +1,23 @@
 import 'package:canvas_test/canvas_test.dart';
 import 'package:flame/extensions.dart';
+import 'package:mocktail/mocktail.dart';
 import 'package:test/test.dart';
 
 void main() {
   group('Canvas extensions tests', () {
```

Comment:
It would be great if you can change the name of this, and the other groups, too.
Just like I described on the new group that you created.